apps/examples/testcase/ta_tc/task_manager/itc/itc_task_manager_main.c

```diff
@@ -104,38 +110,72 @@ void unregister_tasks_4(int gid)
 	int ret;
 
 	group_list_info = (tm_appinfo_list_t *)task_manager_getinfo_with_group(gid, TM_RESPONSE_WAIT_INF);
-	TC_ASSERT_NEQ("task_manager_getinfo_with_group", group_list_info, NULL);
+	if (group_list_info == NULL) {
+		printf("ERROR: failed task_manager_getinfo_with_group in unregister_tasks_4\n");
+		return;
+	}
 
 	while (group_list_info != NULL) {
+
 		task_info = &(group_list_info->task);
+		ret = task_manager_stop(task_info->handle, TM_RESPONSE_WAIT_INF);
+		if (ret != OK && ret != TM_ALREADY_STOPPED_APP) {
+			printf("ERROR: failed task_manager_stop in unregister_tasks_4 ERR: %d\n", ret);
+		}
+
 		ret = task_manager_unregister(task_info->handle, TM_RESPONSE_WAIT_INF);
-		TC_ASSERT_EQ("task_manager_unregister", ret, OK);
+		if (ret != OK) {
+			printf("ERROR: failed task_manager_unregister in unregister_tasks_4 ERR: %d\n", ret);
+		}
 
 		task_manager_clean_info(&task_info);
-		TC_ASSERT_NEQ("task_manager_clean_info", (tm_appinfo_t *)&task_info, NULL);
+		if ((tm_appinfo_t *)&task_info == NULL) {
+			printf("ERROR: failed task_manager_clean_info in unregister_tasks_4\n");
+		}
 
 		group_list_info = group_list_info->next;
 	}
 
 	task_manager_clean_infolist(&group_list_info);
-	TC_ASSERT_NEQ("task_manager_clean_infolist", (tm_appinfo_list_t *)&group_list_info, NULL);
+	if ((tm_appinfo_list_t *)&group_list_info == NULL) {
+		printf("ERROR: failed task_manager_clean_infolist in unregister_tasks_4\n");
+		return;
+	}
 }
 
-void itc_taskmanager_check_dedicate_permission_p(void)
+void unregister_tasks_5(int tm_sample_handle2, int tm_sample_handle3)
```

Comment:
what is task5?

Reply:
oh this file is `itc_task_manger_xx`. You don't need to modify here because this file will be deleted soon. Please consider `itc_taskmanager_xx` for this comment and above.

Reply:
unregister_tasks_5 unregisters the tasks with handles tm_sample_handle, tm_sample_handle2 and tm_sample_handle3